pkg/management/component.go

```diff
@@ -673,27 +689,27 @@ func (c *component) UpdateUser(ctx context.Context, realmName, userID string, us
 	}
 
 	if userInfosUpdated {
-		if keycloakb.IsUpdated(user.BirthLocation, oldDbUser.BirthLocation) {
+		if fieldsComparator.IsAnyFieldUpdated(fields.BirthLocation) {
```

Comment:
Do we have a reason to not use CompareValues() which would perform the update directly ?

Reply:
CompareValues is meant to build add a values to compare in the comparator. I think this is the right method to call. For more clarity, I will call IsFieldUpdated